config/dev.go

```diff
@@ -111,6 +111,35 @@ func SetupVault(addr, rootToken string) error {
 		return err
 	}
 
+	// write sample bulletins
+	if _, err := client.Logical().Write("secret/bulletins/bulletina", map[string]interface{}{
+		"message":    "hello world",
+		"title":    "sampleBulletinA",
+		"type":  "is-success",
```

Comment:
I really should get an automated `go fmt` going on, but for the time being, let's keep it a little cleaner by matching where the double quotes start. `hello world`, `sampleBulletinA`, and `is-success` should match vertically